### PR TITLE
Improve wording of inspect-container man page

### PR DESCRIPTION
**Pull request by cornelius**

### CODE REVIEW

File: man/machinery-inspect-docker.1.md [hunk 1]
```diff
@@ -95,14 +95,14 @@ trigger errors.
 
 ### EXAMPLES
 
-  * Inspect docker-container `mycontainer` and save system description under name 'MyContainer':
+  * Inspect Docker container `mycontainer` and save system description under name 'MyContainer':
```
Comment (mauromorales) [inline man/machinery-inspect-docker.1.md]: What about renaming `mycontainer` to `myimage` ?

File: man/machinery-inspect-docker.1.md [hunk 2]
```diff
@@ -95,14 +95,14 @@ trigger errors.
...
 
     $ `machinery` inspect-container --name=MyContainer mycontainer
```
Comment (mauromorales) [inline man/machinery-inspect-docker.1.md]: probably same here

File: man/machinery-inspect-docker.1.md [hunk 3]
```diff
@@ -95,14 +95,14 @@ trigger errors.
...
 
-  * Inspect docker-container `076f46c1bef1` and save system description under name 'MySecondContainer':
+  * Inspect Docker container `076f46c1bef1` and save system description under name 'MySecondContainer':
```
Comment (mauromorales) [inline man/machinery-inspect-docker.1.md]: and here

File: man/machinery-inspect-docker.1.md (hunk 1 above)
Comment (mschnitzer) [inline man/machinery-inspect-docker.1.md]: I agree here. @mauromorales

**Comment by mschnitzer** [comment]
lgtm from my side when `mycontainer` got renamed to `myimage`

**Comment by mschnitzer** [comment]
lgtm

**Comment by mauromorales** [comment]
lgtm

**Pull request merged**